src/CppParser/Parser.cpp

```diff
@@ -3992,4 +3992,4 @@ Declaration* Parser::GetDeclarationFromFriend(clang::NamedDecl* FriendDecl)
         Decl->LineNumberEnd = MinLineNumberEnd;
     }
     return Decl;
-}
\ No newline at end of file
+}
```

Comment:
Please revert this part. I understand the new line's been silently added by the editor but we use a different editor so if we leave this in, it will keep switching back and forth between commits.